src/components/Reusable/Icon.js

```diff
@@ -0,0 +1,14 @@
+import { useTheme } from "../../context/theme-context"
+import { getIconColor } from '../../utils'
+
+const Icon = (props) => {
+    const { theme } = useTheme()
+
+    return (
+        <span id={props.id} className={`material-icons ${getIconColor(theme)} ${props.classes}`} >
+            {props.children}
```

Comment:
Nitpicking : You can use destructuring here.

Reply:
thanks @Maianki . i have made the changes.